File: Foundation/Number/DTAB.java
import java.util.Scanner;
public class DTAB
{
    public static Scanner scn = new Scanner(System.in);
    public static int decimalToAnyBase(int n, int b)
    {
        int res = 0, pow = 1;
        while(n != 0)
        {
            int rem = n % b;
            n /= b;

            res += rem * pow;
            pow *= 10;
        }
        return res;
    }

    public static void main(String[] args)
    {
        int n = scn.nextInt();
        int b = scn.nextInt();

        System.out.print(decimalToAnyBase(n, b));
    }
}

File: Foundation/Number/DTAB.cpp
#include<iostream>
using namespace std;

int decimalToAnyBase(int n, int b)
{
    int pow = 1, res = 0;
    while(n != 0)
    {
        int rem = n % b;
        n /= b;
        res += rem * pow;
        pow *= 10;
    }
    return res;
}

int main()
{
    int n, b;
    cin>>n;
    cin>>b;
    cout<<decimalToAnyBase(n, b);
    return 0;
}